### ISSUE TITLE
Print Html Horizontally 

### ISSUE THREAD

**Issue by haithem-rihane** [qzind/qz-print#87]
Hi,

I'm using printHTML() function to print html div content on receipt printer 53mm,
I have a problem when printing html content it print vertically so it cut information 
i would like to print it horizontally.

this is html content that i want to print (it contain 2 adjacent table and a text)

<div class="tabContent">
            <div style="float:left">
                <table class="tab1">
                    <tr>
                        <td>text</td>
                        <td></td>
                    </tr>
                    <tr>
                        <td>text</td>
                        <td>-</td>
                    </tr>
                    <tr>
                        <td>text</td>
                        <td>-</td>
                    </tr>
                    <tr>
                        <td>text</td>
                        <td>-</td>
                    </tr>
                    <tr>
                        <td>text</td>
                        <td>-</td>
                    </tr>
                    <tr>
                        <td>text</td>
                        <td>-</td>
                    </tr>
                </table>
            </div>

```
        <div style="float:left">
            <table border="1" class="tab2">
                <tr>
                    <th colspan="6">text</th>
                </tr>
                <tr align="center">
                    <td colspan="2">text</td>
                    <td colspan="2">text</td>
                    <td colspan="2">text</td>
                </tr>
                <tr align="center">
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                </tr>
                <tr align="center">
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                </tr>
                <tr align="center">
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                </tr>
                <tr align="center">
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                </tr>
                <tr align="center">
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">text</td>
                    <td width="50px">-</td>
                    <td width="50px">-</td>
                    <td width="50px">-</td>
                </tr>
                <tr align="center">
                    <td colspan="2">-</td>
                    <td colspan="2">-</td>
                    <td colspan="2">-</td>
                </tr>
            </table>
        </div>           
        <table class="tab3">
            <tr><td>text</td></tr>                    
        </table>
    </div>
```

**Comment by tresf** [comment]
This probably won't work for a while due to our lack of landscape support with HTML content.  Perhaps this one-off release will remedy the situation until we have proper HTML support finished (targeted for December 1st)

https://github.com/tresf/qz-print/releases/tag/v1.9.3-landscape

**Comment by haithem-rihane** [comment]
thx so i will be using pdf till the December 1st

**Comment by haithem-rihane** [comment]
is there line to add to PrintHTML function or automatically print landscape ?

thx

**Comment by tresf** [comment]
> is there line to add to PrintHTML function or automatically print landscape ?

Can you rephrase this question?  Are you talking about the one-off build?  If so, it is here:  https://github.com/qzind/qz-print/commit/5bb7c1c141fe071e065e4bf3c1c2fe2536aca5b1#diff-427d8ce4e50463c43a4eb480f855cd3fR143

-Tres

**Comment by haithem-rihane** [comment]
thx for clarification that waht i need just more info about (String m_orientationString)  content text

**Comment by tresf** [comment]
@haithem-rihane we'll leave this bug open since it will eventually be addressed with our 2.0 release.  Thanks for posting here!

**Comment by tresf** [comment]
Related #93

**Comment by akberenz** [comment]
HTML landscape printing is supported in 2.0

**Comment by haithem-rihane** [comment]
Thx guys great job !!!
